### PR TITLE
Task ys/j265 change podcast live status functionality

### PR DESCRIPTION
**Pull request by yaroslav207**

### CODE REVIEW

File: frontend/src/components/episode/episode.tsx [hunk 1]
```diff
@@ -120,6 +121,17 @@ const Episode: React.FC = () => {
     return <Loader />;
   }
 
+  const handleClickStartLive = (): void => {
+    if (episode) {
+      dispatch(episodeActions.changeEpisodeStatus({
+        ...episode,
+        status: EpisodeStatus.LIVE,
```
Comment (what1s1ove) [inline frontend/src/components/episode/episode.tsx]: according to the task: 
`Live` status can be selected only on the live page (by the button) "Go to Live"
**on the live page**

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/episode-live/episode-live.tsx [hunk 2]
```diff
@@ -3,12 +3,13 @@ import {
   record as recordActions,
   episode as episodeActions,
```
Comment (what1s1ove) [inline frontend/src/components/episode-live/episode-live.tsx]:
```suggestion
```
remove it
we can use only `recordActions` here

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/episode-live/episode-live.tsx (hunk 2 above)
Comment (yaroslav207) [reply frontend/src/components/episode-live/episode-live.tsx]: > remove it
> we can use only `recordActions` here

functionality of episode loading, changing episode status and loading episode comments move to record action?

File: frontend/src/store/record/common.ts [hunk 3]
```diff
@@ -6,6 +6,18 @@ enum ActionType {
   STOP_RECORD = 'recordAudio/stop-record',
   RESET_STATE = 'recordAudio/reset-state',
   SET_LIVE_STREAM = 'recordAudio/set-live-stream',
+  LOAD_EPISODE = 'live-episodes/load-episode',
+  LOAD_COMMENTS_BY_EPISODE_ID = 'live-episodes-comments/load-comments-by-episode-id',
```
Comment (what1s1ove) [inline frontend/src/store/record/common.ts]:
```suggestion
  LOAD_EPISODE = 'recordAudio/load-episode',
```
for all

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/store/record/common.ts (hunk 3 above)
Comment (what1s1ove) [reply frontend/src/store/record/common.ts]: don't fixed

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**